pr-schedule-tests

```diff
@@ -75,6 +75,16 @@ if [ $(echo $CONFIG_LINE | tr ';' '\n' | grep SCRAM_ARCH= | wc -l) -eq 1 ] ; the
     fi
     echo "REQUIRED_TEST=${REQUIRED_TEST}" >> $OUTPUT_FILE
     echo "DRY_RUN=${DRY_RUN}" >> $OUTPUT_FILE
+
+    GPU_LIST=$(paste -sd, "${CMS_BOT_DIR}/gpu_flavors.txt" | tr '[:lower:]' '[:upper:]')
+    NVIDIA_LIST=$(grep '^nvidia_' "${CMS_BOT_DIR}/gpu_flavors.txt" | tr '[:lower:]' '[:upper:]' | paste -sd, -)
+    AMD_LIST=$(grep '^amd_' "${CMS_BOT_DIR}/gpu_flavors.txt" | tr '[:lower:]' '[:upper:]' | paste -sd, -)
+
+    OUTPUT="$ENABLE_BOT_TESTS"
+    OUTPUT="${OUTPUT//GPU/$GPU_LIST}"
```

Comment:
@iarspider , what will happen if in future we add a different test with GPU keyword in it? May be we should use `sed` instead parameter expansion?

Reply:
Ok